pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/DelayedDeliveryTest.java

```diff
@@ -278,7 +278,6 @@ public void testDelayedDeliveryWithMultipleConcurrentReadEntries()
         for (int i = 0; i < N; i++) {
             msg = consumer.receive(10, TimeUnit.SECONDS);
             receivedMsgs.add(msg.getValue());
-            consumer.acknowledge(msg);
```

Comment:
Why need to remove this line?

Reply:
I added that line in the PR that created this problem.

this line made the test pass,
with this "fix" now the test still passes

this test is very interesting because it runs many readMoreEntries() in a separate thread, simulating the mess that happens inside the broker

it is important to revert this change to the test

Reply:
I see.